### PR TITLE
Make it possible to configure the color of the social icons

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
This adds the option to configure the color of social links in the navigation via theme.json.

I'm not 100% sure that this is the right place to put this in the theme.json file - it might make more sense to put it under navigation...

### CODE REVIEW

File: blockbase/theme.json [hunk 1]
```diff
@@ -294,6 +294,11 @@
 				"opacity": 1,
 				"width": "150px"
 			},
+			"social-links": {
```
Comment (scruffian) [inline blockbase/theme.json]: This is just for testing

Comment (MaggieCabrera) [reply blockbase/theme.json]: this is only being applied to social links inside navigation blocks so maybe we'd better change the custom variable name to reflect that?

Comment (scruffian) [reply blockbase/theme.json]: good idea

**Review by MaggieCabrera — APPROVED** [review]
besides deciding on the name of the variable, this works for me.

File: blockbase/theme.json (hunk 1 above)
Comment (scruffian) [reply blockbase/theme.json]: changed to navigaiton/social-links, what do you think?

Comment (MaggieCabrera) [reply blockbase/theme.json]: yeah that sounds about right!

**Pull request merged**